ocs_ci/utility/rhv.py

```diff
@@ -465,3 +465,32 @@ def stop_rhv_vms(self, vms, timeout=600, force=False):
             except TimeoutExpiredError:
                 logger.error(f"RHV VM {vm.name} is still Running")
                 raise
+
+    def start_rhv_vms(self, vms, wait=True, timeout=600):
+        """
+         Run the RHV virtual machines
+
+        Args:
+            vms (list): list of RHV vm instances
+            wait (bool): Wait for RHV VMs to start
+            timeout (int): time in seconds to wait for VM to reach 'up' status.
+
+        """
+        for vm in vms:
+            # Find the virtual machine
+            vm_service = self.get_vm_service(vm.id)
+            vm_service.start(wait=wait)
+            if wait:
```

Comment:
in order to make the start VMs go faster, instead of starting and waiting for each VM, we can first start all VMs and then wait for all of them.

Reply:
incorporated the review  comments